inf/src/main/java/config/ServerConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package config;
+
+import java.util.Map;
+
+public class ServerConfiguration {
+
+    private static final String LOG_MESSAGE_KEY = "logMessage";
+    private static final String PACKAGE_NAME_KEY = "packageName";
+    private static final String PORT_KEY = "port";
+    private static final String HOST_KEY = "host";
+
+    private String logMessage;
+    private String packageName;
+    private int port;
+    private String host;
+
+
+    public ServerConfiguration (Map<String, Object> jsonMap) {
+        this.logMessage = (String) jsonMap.get(LOG_MESSAGE_KEY);
```

Comment:
Again, null check for required configuration params

Reply:
done